src/auth.js

```diff
@@ -0,0 +1,51 @@
+import jsonwebtoken from "jsonwebtoken";
+
+const USERNAME = "user";
+const PASSWORD = "password";
+const tokens = [];
+const JWT_SECRET = "blahblahblahblahBLAHBLAHBLAHBLAHBLAHBLAHBLAHBLAHBLAAAA";
+const FIVE_MINS = 5 * 60 * 1000;
+
+export const login = (req, res) => {
+	const authHeader = req.headers.authentication;
+  const b64creds = authHeader.slice(6);
```

Comment:
The tab width is inconsistent here. Some lines using two spaces, others use tabs.